listenbrainz_spark/stats/user.py

```diff
@@ -0,0 +1,62 @@
+import sys
+import listenbrainz_spark
+import time
+from datetime import datetime  
+
+
+def top_artist(user_name):
+	playcounts_df = listenbrainz_spark.sql_context.sql("""
+            SELECT artist_name, count(track_name) as cnt
+              FROM listen
+             WHERE user_name = '%s'
+          GROUP BY artist_name
+          ORDER BY cnt DESC
+             LIMIT 20
+        """ % user_name)
+	playcounts_df.show()
+
+def top_recording(user_name):
+	playcounts_df = listenbrainz_spark.sql_context.sql("""
+            SELECT track_name, count(track_name) as cnt
+              FROM listen
+             WHERE user_name = '%s'
+          GROUP BY track_name
+          ORDER BY cnt DESC
+             LIMIT 20
+        """ % user_name)
+	playcounts_df.show()
+
+def top_release(user_name):
+	playcounts_df = listenbrainz_spark.sql_context.sql("""
+            SELECT release_name, count(release_name) as cnt
```

Comment:
we should get as much useful data in here as possible. See queries here (https://github.com/metabrainz/listenbrainz-server/blob/master/listenbrainz/stats/user.py)